dbus/org.jfhbrook.plusdeck.conf

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?> <!-- -*- XML -*- -->
+
+<!DOCTYPE busconfig PUBLIC
+ "-//freedesktop//DTD D-BUS Bus Configuration 1.0//EN"
+ "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd">
+<busconfig>
+  <policy user="root">
+    <allow own="org.jfhbrook.plusdeck"/>
+  </policy>
+
+  <policy context="default">
+    <allow send_destination="org.jfhbrook.plusdeck" user="root"/>
+    <allow send_destination="org.jfhbrook.plusdeck" group="plusdeck"/>
```

Comment:
This needs to be tested by actually creating the group and adding my user to it.